src/actions/queueaction/test_queueaction.ts

```diff
@@ -0,0 +1,39 @@
+import * as chai from "chai"
+import * as Hub from "../../hub"
+
+import {ExecuteProcessQueue} from "../../../lib/server/action_process_queue"
+import {QueueAction} from "./queueaction"
+
+const action = new QueueAction()
+const queue = new ExecuteProcessQueue()
+process.execArgv = ["./node_modules/ts-node/dist/bin.js"]
+process.argv = ["node", "./integrations/src/boot.ts"]
+process.env.CHILD_TEST = "true"
```

Comment:
how come all these env vars have to be set?

Reply:
The process is running with mocha and a different context, it just didn't work without changing the process vars. The process env was just convenient and I saw an example of it being used for testing elsewhere so it made sense

Reply:
Do you know what's looking at these env vars? Would just be good to have a comment explaining it if nothing else. Is it something about how `fork` works?

Reply:
figuring out this stuff was a pain realizing how much has to be made more close to how it runs normally than it would as a test_fixture